app/services/framework_nomis_mappings/importer.rb

```diff
@@ -89,5 +91,16 @@ def nomis_code_ids_to_mappings
     def responses_to_codes
       @responses_to_codes ||= framework_responses.joins(:framework_nomis_codes).select('framework_responses.id as id, framework_nomis_codes.id as nomis_code_id').group_by(&:id)
     end
+
+    def log_exception(description, params)
+      Raven.capture_message(
+        description,
+        extra: {
+          id: person_escort_record.id,
```

Comment:
Could `person_escort_record` ever be nil her? I see in the initialize method it is possible. So maybe use `person_escort_record&.id` instead ?